packages/shared/src/keys.ts

```diff
@@ -39,7 +39,12 @@ export function parsePublishableKey(
   key = key || '';
 
   if (!key || !isPublishableKey(key)) {
-    if (options.fatal) {
+    if (options.fatal && !key) {
+      throw new Error(
+        'Publishable key is missing. Ensure that your publishable key is correctly configured. Make sure to double-check your environment configuration (.env file).',
```

Comment:
```suggestion
        (.env file).',
```

Not sure if we need to specify the file here as it can be a bit different depending on platform/framework etc.
example: Cloudflare Wrangler locally reads from `.dev.vars`

Reply:
Removed it!